Desktop/po/jaspDesktop-zh_Hant.po

```diff
@@ -1782,230 +1781,246 @@ msgstr "圖"
 
 msgctxt "AnalysisForm|"
 msgid "Collection"
-msgstr ""
+msgstr "集合"
 
 msgctxt "AnalysisForm|"
 msgid "Result"
-msgstr ""
+msgstr "結果"
 
 msgctxt "AnalysisForm|"
 msgid "Output"
-msgstr ""
+msgstr "報表"
 
 msgctxt "AnalysisForm|"
 msgid "Input"
-msgstr ""
+msgstr "輸入"
 
 msgctxt "JASPControl|"
 msgid "Option"
-msgstr ""
+msgstr "選項"
 
 msgctxt "JASPControl|"
 msgid "Section"
-msgstr ""
+msgstr "部分"
 
 msgctxt "JASPControl|"
 msgid "CheckBox"
-msgstr ""
+msgstr "核取方塊"
 
 msgctxt "JASPControl|"
 msgid "Switch"
-msgstr ""
+msgstr "開關"
 
+#, fuzzy
 msgctxt "JASPControl|"
 msgid "Entry Field"
-msgstr ""
+msgstr "輸入欄位"
 
+#, fuzzy
 msgctxt "JASPControl|"
 msgid "Radio Button"
-msgstr ""
+msgstr "單選按鈕"
 
+#, fuzzy
 msgctxt "JASPControl|"
 msgid "Radio Buttons"
-msgstr ""
+msgstr "單選按鈕"
 
 msgctxt "JASPControl|"
 msgid "Variables"
-msgstr ""
+msgstr "變數"
 
 msgctxt "JASPControl|"
 msgid "ComboBox"
-msgstr ""
+msgstr "下拉式組合方塊"
 
+#, fuzzy
 msgctxt "JASPControl|"
 msgid "Input ListView"
-msgstr ""
+msgstr "輸入清單檢視"
 
 msgctxt "JASPControl|"
 msgid "TableView"
-msgstr ""
+msgstr "表格方式檢視"
 
+#, fuzzy
 msgctxt "JASPControl|"
 msgid "Slider"
-msgstr ""
+msgstr "滑塊"
 
 msgctxt "JASPControl|"
 msgid "TextArea"
-msgstr ""
+msgstr "文字區"
 
+#, fuzzy
 msgctxt "JASPControl|"
 msgid "Button"
-msgstr ""
+msgstr "按鈕"
 
+#, fuzzy
 msgctxt "JASPControl|"
 msgid "Factors Form"
-msgstr ""
+msgstr "因素形式"
 
+#, fuzzy
 msgctxt "JASPControl|"
 msgid "List of Components"
-msgstr ""
+msgstr "元件清單"
 
 msgctxt "JASPControl|"
 msgid "Group Box"
-msgstr ""
+msgstr "組箱"
 
 msgctxt "JASPDoubleValidator|"
 msgid "The value must be "
-msgstr ""
+msgstr "該值必須為 "
 
-#, qt-format
+#, fuzzy, qt-format
 msgctxt "JASPDoubleValidator|"
 msgid "&#8805; %1"
-msgstr ""
+msgstr "≥ %1"
 
 #, qt-format
 msgctxt "JASPDoubleValidator|"
 msgid "&gt; %1"
-msgstr ""
+msgstr "> %1"
 
 msgctxt "JASPDoubleValidator|"
 msgid " and "
-msgstr ""
+msgstr " 與 "
 
-#, qt-format
+#, fuzzy, qt-format
 msgctxt "JASPDoubleValidator|"
 msgid "&#8804; %1"
-msgstr ""
+msgstr "≤ %1"
 
 #, qt-format
```

Comment:
thank you for your all effort @changyahu, I don't know if this would be hurt that translator using '< > ≤...' rather than HTML symbol...(so I had changed it back on Weblate )

@boutinb  but I think it's better to remove `tr()` [here](https://github.com/jasp-stats/jasp-desktop/blob/583604e1b1b0ebb15602b46014cac0cab0979a9c/QMLComponents/controls/jaspdoublevalidator.cpp#L80-L101) right? it's no sense  translate a symbol and placeholder:-)